src/Http/Controllers/V1/Shop/Customer/CartController.php

```diff
@@ -92,6 +100,16 @@ public function update(Request $request, CartItemRepository $cartItemRepository)
         foreach ($request->qty as $itemId => $qty) {
             $item = $cartItemRepository->findOneByField('id', $itemId);
 
+            if (
+                $item->type == 'downloadable' 
+                || $item->type ==  'bundle' 
+                || $item->type == 'booking'
+            ) {
+                return response([
+                    'message' => __('rest-api::app.common-response.error.cannot-update-product', ['name' =>  $item->type . ' ' .  __('rest-api::app.common-response.general.product-quantity')]),
+                ]);
+            }
```

Comment:
Need to check.

Reply:
I the core, the cart quantity increment / decrement button is disabled for this types of product on checkout page.

Reply:
Removed